flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -172,7 +172,7 @@ under the License.
 		<!-- A planner dependency won't be necessary once FLIP-32 has been completed. -->
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-table-planner_${scala.binary.version}</artifactId>
+			<artifactId>flink-table-planner-blink_${scala.binary.version}</artifactId>
```

Comment:
Why just change cassandra?
Maybe scope to test and remove above comments?

Reply:
I'm not sure about this, did you see any problem with it?

Reply:
It is not a big deal.